pages/pipelines/command_step.md

```diff
@@ -306,6 +306,24 @@ _Optional Attributes_
       <em>Example:</em> <code>2</code>
     </td>
   </tr>
+  <tr>
+    <td><code>signal</code></td>
+    <td>
+      The signal sent to a process, if the process did not intercept and handle the signal<br>
+      <em>Example:</em> <code>"*"</code><br>
+      <em>Example:</em> <code>kill</code><br>
+      <em>Example:</em> <code>SIGINT</code>
```

Comment:
What do you think of listing out the example values rather than repeating? Thinking:

```suggestion
      <p><em>Examples:</em>
      <ul>
        <li><code>"*"</code></li>
        <li><code>kill</code></li>
        <li><code>SIGINT</code></li>
      </ul></p>
```

Reply:
Sure! I was following the pattern on the page.

Reply:
Yeah, that would be awesome 🙌🏻